src/FakeItEasy/Configuration/IThenConfiguration.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace FakeItEasy.Configuration
+{
+    using System.Diagnostics.CodeAnalysis;
+
+    /// <summary>
+    /// Configures the next behavior for this call.
+    /// </summary>
+    /// <typeparam name="TInterface">The type of configuration interface to return.</typeparam>
+    public interface IThenConfiguration<out TInterface>

```

Comment:
I think put them in a separate PR. The change isn't really related to this issue, and it might make the change (not that anyone will notice) easier to spot if it's a separate issue and line in the release notes.
Also, it gets this PR in faster!